webapp/components/Editor/Editor.vue

```diff
@@ -107,29 +106,7 @@ export default {
       return extensions
     },
   },
-  watch: {
-    value: {
-      immediate: true,
-      handler: function(content, old) {
-        const contentHash = stringHash(content)
-        if (!content || contentHash === this.lastValueHash) {
-          return
-        }
-        this.lastValueHash = contentHash
-        this.$nextTick(() => this.editor.setContent(content))
-      },
-    },
-    placeholder: {
-      immediate: true,
-      handler: function(val) {
-        if (!val || !this.editor) {
-          return
-        }
-        this.editor.extensions.options.placeholder.emptyNodeText = val
-      },
-    },
-  },
-  created() {
```

Comment:
code creep? @roschaefer ... why were these changes made in this context? 
this has the undesired side effect that makes the placeholder text not reactive, but also now instead of just approving and merging, we need to discuss what these changes have anything to do with fixing the server side-rendering bug

Reply:
sorry, I reverted your changes @roschaefer because I know that you are busy today with your talk, and I'd like to get this bug fix in today's deploy. Normally, I would like to discuss this before making the decision, but under the circumstances, I felt it more important to get the bug fix in....